process/logsevents/nftsProcessor.go

```diff
@@ -52,31 +55,55 @@ func (np *nftsProcessor) processLogAndEventsNFTs(
 		return tokens, tagsCount
 	}
 
-	for _, txLog := range logsAndEvents {
+	for logHash, txLog := range logsAndEvents {
 		if check.IfNil(txLog) {
 			continue
 		}
 
-		np.processNFTOperationLog(txLog, accounts, tokens, timestamp, tagsCount)
+		np.processNFTOperationLog(logHash, txLog, accounts, tokens, timestamp, tagsCount, txsMap, scrsMap)
 	}
 
 	return tokens, tagsCount
 }
 
 func (np *nftsProcessor) processNFTOperationLog(
+	logHash string,
 	txLog nodeData.LogHandler,
 	accounts data.AlteredAccountsHandler,
 	tokens data.TokensHandler,
 	timestamp uint64,
 	tagsCount tags.CountTags,
+	txsMap map[string]*data.Transaction,
+	scrsMap map[string]*data.ScResult,
 ) {
 	events := txLog.GetLogEvents()
 	if len(events) == 0 {
 		return
 	}
 
+	logHashHexEncoded := hex.EncodeToString([]byte(logHash))
```

Comment:
duplicated code with the fungibleESDTProcessor.go

Reply:
done.